cost/aws/savings_realized/aws_savings_realized.pt

```diff
@@ -257,9 +328,12 @@ script "js_get_savings_realized", type: "javascript" do
 
     //Savings Plan savings
     _.each(sp_costs, function(sp){
-      if (sp.instance_type == inst_hr.instance_type && sp.region == inst_hr.region){
+      if (sp.instance_type == inst_hr.instance_type && sp.region == inst_hr.region && sp.operating_system == inst_hr.operating_system &&
+          sp.license_model == inst_hr.license_model && sp.database_edition == inst_hr.database_edition &&
+          sp.database_engine == inst_hr.database_engine && sp.deployment_option == inst_hr.deployment_option){
+        sp_processed_counter++
         savings_realized_per_inst_hr = inst_hr.avg_od_cost_inst_hr - (sp.cost / sp.usage_amount)
-        console.log(savings_realized_per_inst_hr, sp.region, sp.instance_type)
+        //console.log(savings_realized_per_inst_hr, sp.region, sp.instance_type)
```

Comment:
Is this commented code really necessary?

Reply:
Really  it is not, It's just for testing/debug. I'll remove all the commented console.log lines. Good catch, thanks